tools/congestion-model/src/main.rs

```diff
@@ -161,9 +167,10 @@ fn strategy(strategy_name: &str, num_shards: usize) -> Vec<Box<dyn CongestionStr
             "No queues" => Box::new(NoQueueShard {}) as Box<dyn CongestionStrategy>,
             "Global TX stop" => Box::<GlobalTxStopShard>::default(),
             "Simple backpressure" => Box::<SimpleBackpressure>::default(),
-            "Fancy Global Transaction Stop" => Box::<FancyGlobalTransactionStop>::default(),
+            "Fancy Stop" => Box::<NepStrategy>::default(),
```

Comment:
Is this intentionally mapping to NepStrategy?

Reply:
Oh no good catch thanks! I was copy pasting files and renaming things and this sneaked in.